ignore/src/dir.rs

```diff
@@ -513,6 +525,14 @@ impl IgnoreBuilder {
         self
     }
 
+    /// Enables reading of ignore files in addition to `.ignore`
+    ///
+    /// Earlier names have lower precedence
+    pub fn ignorefile(&mut self, filename: String) -> &mut IgnoreBuilder {
```

Comment:
`ignorefile()` vs existing `add_ignore()` (for global ignorefiles) might be a bit confusing. Happy to change the name into something better. Or maybe change `add_ignore` to `add_global_ignore`?